test/index.js

```diff
@@ -1,10 +1,13 @@
 'use strict';
 
+const Fs = require('fs');
 const Path = require('path');
+const { expect } = require('code');
+const { graphql } = require('graphi');
 const Hapi = require('hapi');
 const Lab = require('lab');
 const CloudApiGql = require('../lib/');
-
+const schema = Fs.readFileSync(Path.join(__dirname, '../lib/schema.graphql'));
```

Comment:
It's not a big deal, but if you use `Path.resolve()` here, it will get rid of the `'..'` step for you.